File: calendar1_R&A_Algorithm/HPN C# R&A CODE/HpnCalendarTools/HpnCalendarTools/hpn/settings/DateString.cs
using System;

namespace hpn.settings
{
    public static class DateString
    {
        private static String format = "dd.MM.yyyy HH:mm:ss";

        public static String Format
        {
            get
            {
                
                return format;
            }
            set
            {
                format = value;
            }
        }

    }
}

File: calendar1_R&A_Algorithm/HPN JAVA R&A CODE/HpnCalendarTools/src/hpn/settings/DateString.java
package hpn.settings;

import java.text.SimpleDateFormat;
import java.util.Date;

public final class DateString
{
    private static String format = "dd.MM.yyyy HH:mm:ss";

    public static String Format()
    {
       return format;
    }
    
    public static String dateToString(Date date)
    {
    	SimpleDateFormat dateTimeFormat = new SimpleDateFormat (format);
		return dateTimeFormat.format(date);
    }
}
